app/src/main/java/com/globant/practice/di/PracticeModule.java

```diff
@@ -71,6 +72,17 @@ SubscriberDetailsPresenter provideSubscriberDetailsPresenter(FetchSubscriberProf
     }
 
     /**
+     * Returns a unique reference of the WebClientPresenter
+     *
+     * @return WebClientPresenter reference
+     */
+    @Provides
+    @Singleton
+    WebClientPresenter provideWebClientPresenter() {
```

Comment:
Since you're injecting the constuctor of each Presenter, there is no need in providing them. You can remove them.